views/js/controller/Delivery/index.js

```diff
@@ -42,31 +65,107 @@ define([
      */
     var cssScope = '.delivery-index';
 
+    var serviceUrl = urlHelper.route('deliveries', 'DeliverySelection', 'taoProctoring');
+    var pauseUrl = urlHelper.route('pauseExecutions', 'Monitor', 'taoProctoring');
+    var sessionsUrl = urlHelper.route('deliveryExecutions', 'Monitor', 'taoProctoring');
+
+    /**
+     * Filters the disconnection errors
+     * @param {Error} err
+     */
+    function handleOnDisconnect(err) {
+        if (err.code === 403) {
+            //we just leave if any 403 occurs
+            window.location.reload(true);
+        }
+    }
+
+    /**
+     * Gets a list of named sessions
+     * @param {Array} sessions
+     * @returns {Array}
+     */
+    function getSessionsNames(sessions) {
+        return _.map(sessions, function (session) {
+            var testTaker = session.testTaker;
+            return {
+                id: session.id,
+                label: testTaker.firstName + ' ' + testTaker.lastName
+            };
+        });
+    }
+
+    /**
+     * Gets the list of sessions for a particular delivery
+     * @param {String} deliveryId
+     * @returns {Promise}
+     */
+    function getDeliverySessions(deliveryId) {
+        return new Promise(function(resolve, reject) {
+            $.ajax({
```

Comment:
Well... My bad. I started to work on proxy after having made this. I just updated it, but it depends on a fix in [taoCore#1224](https://github.com/oat-sa/tao-core/pull/1224)